package.json

```diff
@@ -32,7 +32,9 @@
         "coc.nvim"
     ],
     "activationEvents": [
-        "onLanguage:ps1"
+        "onLanguage:ps1",
+        "onLanguage:powershell",
```

Comment:
```suggestion
        "onLanguage:powershell",
        "onLanguage:PowerShell",
        "onLanguage:Powershell",
```

Reply:
I'm fine with this. Though I don't like the last one because it's wrong 😛

But we can keep it since people use that a lot